clevercloud/cron.json

```
@@ -11,6 +11,7 @@
   "*/1 * * * * $ROOT/src/tools/jobs/send-email-candidats.sh",
   "0 0 * * * $ROOT/src/tools/jobs/delete-cv-six-mois.sh",
   "15 0 * * * $ROOT/src/tools/jobs/delete-inactive-accounts.sh",
+  "0 3 * * * $ROOT/src/tools/jobs/rgpd-delete-accounts.sh",
```

Juste pour qu'on s'en souvienne avec le lien avec le CRON du dessus
```suggestion
  "0 3 * * * $ROOT/src/tools/jobs/delete-inactive-candidats.sh",
```